Sources/TuistKit/Utils/Sequence+KeyPath.swift

```diff
@@ -0,0 +1,23 @@
+extension Sequence {
```

Comment:
Same comment here regarding moving the extension to `TuistCore` and making it public.